integration_tests/src/main/scala/org/apache/spark/sql/tests/datagen/bigDataGen.scala

```diff
@@ -1804,23 +1822,40 @@ class DBGen {
    */
   def apply(name: String): TableGen = tables(name.toLowerCase)
 
+  def get(name: String): Option[TableGen] = tables.get(name)
```

Comment:
What made you decide to not convert this to a lower case just like you did in `apply`? Trying to know the thought process

Reply:
I forgot. Good catch.

Reply:
Actually, I understand why you didn't convert it to lower case here. I think what I don't understand is why convert it to lowercase in `apply`

Reply:
Spark by default is case insensitive.  A table names "FOO" and "foo" are the same, which is why I normalize them all to be lowercase. I do that when I add the table to the map too.